BUILDING.md

```diff
@@ -1,244 +1,102 @@
-
-> Note: If you're on a newer Linux distribution using the unified cgroup hierarchy with cgroups v2, you may need to disable it to work with current versions of runc.
-> You'll know this is the case if you see an error like `docker: Error response from daemon: OCI runtime create failed: this version of runc doesn't work on cgroups v2: unknown.`
-> Set the kernel parameter `systemd.unified_cgroup_hierarchy=0` in your boot configuration (e.g. GRUB) and reboot.
-
-### Build process
-
-To build an image, run:
-
-```shell
-cargo make
+You'll also need to enable the containerd-snapshotter and buildkit features for your docker daemon. This is required to ensure docker compatibility with OCI Images (which Kits are stored in).
+```json
+{
+  "features": {
+    "buildkit": true,
+    "containerd-snapshotter": true
+  }
+}
 ```
+#### Crane
+Crane is a container tool that the container registry team at google have published that handles oci image operations cleaner. It does not require a daemon and thus you shouldn't need the above docker features to use it. Twoliter (since 0.4.2) now supports utilizing crane instead of docker if it is installed. 
 
-This will build an image for the default variant (a recent `aws-k8s-*`, see the `BUILDSYS_VARIANT` variable in [Makefile.toml](Makefile.toml) to find the current default variant).
-All packages will be built in turn, and then compiled into an `img` file in the `build/images/` directory.
 
-The version number in [Release.toml](Release.toml) will be used in naming the file, and will be used inside the image as the release version.
-If you're planning on [publishing your build](PUBLISHING.md), you may want to change the version.
+[Installation Instructions](https://github.com/google/go-containerregistry/tree/main/cmd/crane)
 
-To build an image for a different variant, run:
+## Build the Kit
 
-```shell
-cargo make -e BUILDSYS_VARIANT=my-variant-here
+Building the core kit can be done by using the makefile targets.
 ```
-
-To build an image for a different architecture, run:
-
-```shell
-cargo make -e BUILDSYS_ARCH=my-arch-here
+make ARCH=<architecture>
 ```
+The kit build will result in an extracted local kit of the core kit in `build/kits/bottlerocket-core-kit`. Along with the extracted version of the core kit you will see a tar archive inside the same path for the architecture that was built. This is the OCI image generated by twoliter.
 
-If you want to limit the build concurrency, set `BUILDSYS_JOBS` (the default is `8`):
+## Publish the Kit
+After the kit has been built you can then publish the kit image to your private registry. This will allow you to consume it to build and test a variant.
+### Setup a private ECR for development/testing
+It is recommended that you have some form of protected container registry to use for testing. Currently twoliter does not allow the transfer of locally stored kits between a local copy of bottlerocket-core-kit and bottlerocket. 
```

Comment:
nit: I would omit the second sentence, since it forces an awkward choice between using project names vs. the preferred names:
```suggestion
It is recommended that you have some form of protected container registry to use for testing.
```